schema/methods_directory/local_orbital/precision.json

```diff
@@ -0,0 +1,15 @@
+{
+    "schemaId": "methods-directory-local-orbital-precision",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "precision schema for local-orbital dft",
+    "properties": {
+        "energy_convergence": {
+            "description": "SCF convergence control for DFT calculations (i.e. 1e-7)",
+            "type": "string"
```

Comment:
(1) Perhaps we can reuse the "energy" property, part of ESSE already, here, and just include the number instead.
(2) This is often called "tolerance", is it not? Maybe `energySCFTolerance`

Also see [CamelCase vs snake_case](https://capitalizemytitle.com/camel-case/#:~:text=Snake%20Case%20(stylized%20as%20snake_case,(_)%20instead%20of%20a%20space.&text=Python%20uses%20snake%20case%20for,and%20package%20names%2C%20among%20others.) pls. We've been using camel case for the keys in schema, and snake case for the file names. Maybe something to add to README for any future contributors.

Reply:
for nwcem:
convergence = scf convergence control
tolerance = DFT integral evaluation tolerance.

Sorry about the key naming mix up. I went back and fixed all instances so keys are named with camelCase.